src/app_client.rs

```diff
@@ -72,6 +72,7 @@ async fn process_block(
 	app_id: u32,
 	block: &ClientMsg,
 	pp: PublicParameters,
+	thresh: usize,
```

Comment:
We can put threshold into AppClientConfig struct, to avoid cluttering parameter list.